docs/reference/migration/migrate_7_14.asciidoc

```diff
@@ -145,15 +145,28 @@ Discontinue use of the `type` parameter in `geo_bounding_box` queries.
 
 [discrete]
 [[implicitly-disabled-security]]
-.The default behavior of disabling security on basic and trial licenses is deprecated
+.Disabling security on basic and trial licenses by default is deprecated.
```

Comment:
```suggestion
.Having security disabled by default on basic and trial licenses is deprecated.
```

This may help to clarify that we are not deprecating the (user) action to disable security, but the default value. What do you think?